cmake/modules/FindDEAL_II_TRILINOS.cmake

```diff
@@ -48,6 +48,16 @@ find_package(TRILINOS_CONFIG
   )
 
 if(Trilinos_FOUND)
+  # run find_package to populate variables like Kokkos_VERSION not
+  # set by the find_package call to Trilinos:
+  set(CMAKE_CXX_EXTENSIONS OFF)
+  find_package(Kokkos 3.4.0 REQUIRED QUIET
```

Comment:
This `REQUIRED` here is a problem because it forces the configuration to fail if Trilinos does not ship with a sufficient Kokkos version.